pkg/controller/clusterstorage/clusterstorage_controller.go

```diff
@@ -183,7 +184,36 @@ func (r *ReconcileClusterStorage) Reconcile(request reconcile.Request) (reconcil
 		return reconcile.Result{}, err
 	}
 
-	// StorageClass already exists - don't requeue
+	// Check to see if modifications have been made to the StorageClass attributes
+	needUpdate := false
+	newSCC := sc.DeepCopy()
```

Comment:
SCC is a kinda not a great variable name because it has other meaning in context of openshift.

Reply:
I updated this to be `comparisonSC`.